ui/src/api/mixins/ApiResource.ts

```diff
@@ -0,0 +1,24 @@
+import { Constructor } from '@tpluscode/rdfine'
+import { Mixin } from '@tpluscode/rdfine/lib/ResourceFactory'
+
+declare module 'alcaeus/Resources' {
+  export interface HydraResource {
+    clientPath: string
+  }
+}
```

Comment:
This extends the base interface so that the new property is recognised on all instances of `HydraResource`